pallets/registrar/src/lib.rs

```diff
@@ -626,12 +626,15 @@ pub mod pallet {
             para_id: ParaId,
             manager_address: T::AccountId,
         ) -> DispatchResult {
-            let origin = ensure_signed(origin)?;
+            // Allow root to force set para manager.
+            if let Err(_) = ensure_root(origin.clone()) {
+                let origin = ensure_signed(origin)?;
```

Comment:
Maybe let's use `ensure_signed_or_root` here, to keep the pattern of the first line always being `let origin = ...`.

https://paritytech.github.io/polkadot-sdk/master/frame_system/fn.ensure_signed_or_root.html